static/nodejs_server/types.ts

```diff
@@ -41,4 +41,6 @@ export interface TileResult {
     value: number;
     date: string;
   };
+  // The link to the data commons explore page for this result.
+  dcLink?: string;
```

Comment:
nit: dcUrl ?  (to consistently refer to all URLs as such in this json)

Reply:
done